File: C#/Strategy/PaymentStrategys/BankTransferPaymentStrategy.cs
using Strategy.Interfaces;

namespace Strategy.PaymentStrategys;

public sealed class BankTransferPaymentStrategy : IPaymentStrategy
{
    /// <summary>
    /// Bank Transfer payment processing
    /// </summary>
    /// <param name="amount"><inheritdoc/></param>
    public void ProcessPayment(decimal amount)
        => Console.WriteLine($"Processing bank transfer payment for {amount}");
}

File: PHP/Patterns/src/Behavioral/Strategy/PaymentWays/BankTransferPaymentStrategy.php
<?php

namespace DesignPatterns\Behavioral\Strategy\PaymentWays;

use DesignPatterns\Behavioral\Strategy\Interfaces\PaymentStrategy;

class BankTransferPaymentStrategy implements PaymentStrategy
{
    public function processPayment(float $amount): void
    {
         echo "Processing payment of bank transfer...";
    }
}